### ISSUE TITLE
Local file target misses some data on Linux (Ubuntu)

### ISSUE THREAD

**Issue by kamatama41** [tumugi/tumugi#139]
Hi, I encountered that incident, but the target writes the whole data completely on Mac OS X (my laptop). Following snippet is a reproducible example.

**test_task.rb**
```ruby
def prepare
  `rm -rf input.txt output.txt`
  File.open('input.txt', 'w') do |f|
    10000.times do |i|
      f.puts i
    end
  end
end

prepare

task :example do
  run do
    output.open('w') do |file|
      File.open('input.txt', 'r') do |input|
        input.each_line do |line|
          file.puts line
        end
      end
    end
  end

  output target(:local_file, 'output.txt')
end
```

**Run on Ubuntu**
Ruby: 2.3.3
Tumugi: 0.6.3
`wc -l output.txt` is expected to be 10000, but not 10000
```sh
$ uname -a
Linux xxxxxxxx 3.13.0-96-generic #143-Ubuntu SMP Mon Aug 29 20:15:20 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ bundle exec tumugi run -f ./test_task.rb example
...

$ wc -l output.txt
8417 output.txt
```

**Comment by hakobera** [comment]
@kamatama41 Thanks to report. I fix this issue by #140 .
Can you try the branch and confirm your problem fixed?

**Comment by kamatama41** [comment]
It works, thank you for quick response!
